simphony_ui/couple_openfoam_liggghts.py

```diff
@@ -20,6 +22,8 @@ def run_calc(global_settings, openfoam_settings, liggghts_settings):
         The trait model containing the Openfoam parameters
     liggghts_settings : LiggghtsModel
         The trait model containing the Liggghts parameters
+    progress_callback
+        A callback function which will return the progress of the computation
```

Comment:
which will be called with the progress state of the calculation as an integer 0:100